Update how to enable Apport to reflect availability in the Control Center (#577)
ubuntu/ubuntu-project-docs: docs/contributors/debugging/apport.md

(how-to-enable-apport)=
## How to enable Apport

On Ubuntu Desktop, you can manage error reporting from the Control Center. Open {guilabel}`Settings`, go to {guilabel}`Privacy & Security` > {guilabel}`Diagnostics`, and use {guilabel}`Send error reports to Canonical` to choose how error reports are handled.

The default setting is {guilabel}`Manual`, which means Ubuntu asks before sending an error report. You can also choose to send reports automatically or disable error reporting from the same settings page.

> If you are working on a server or another system without a graphical settings interface, you can still inspect crash reports from the command line. Apport stores crash reports in {file}`/var/crash/`, and tools such as {command}`apport-cli` can process pending reports from a terminal.

Reviewer: ```suggestion
To inspect crash reports from the command line, use the {command}`apport-cli` tool. Apport stores crash reports in the {file}`/var/crash/` directory.
```
